pkgs/applications/editors/kile/default.nix

```diff
@@ -48,6 +49,7 @@ let
         kparts
         ktexteditor
         kwindowsystem
+        okular.unwrapped
```

Comment:
You should also add Okular to the `paths` argument of `kdeWrapper` below, otherwise Kile will not find Okular at runtime unless it is installed system-wide.